### ISSUE TITLE
Expand Alert widget

### ISSUE THREAD

**Issue by lookeypl** [LKostyra/LukeBot#4]
Simple solution to responding to Twitch-based events.

LukeBot already has an `Alerts.html` widget, which has to be expanded into a simple Alert box - displaying what type of alert it is, and a potential message.

Some degree of configurability is necessary here, which needs researching on how to tackle that. I think the best strat is the usual - do the first hard-coded alert box first, then use that experience as a base for Widget's configuration.

Bonus points for displaying a picture/GIF.

Blocked by: #3

**Comment by lookeypl** [comment]
Resolved by 08f886b and following commits.

Alerts widget is now hard-coded-configurable based on its name (left alignment instead of right by adding "_Left" suffix to Widget's name). This system is very temporary and will need to be solved via a separate issue.

Alerts now only react to Twitch subscriptions, but can be easily expanded - also a task for later.
